README.md

```diff
@@ -31,6 +31,14 @@ Prior to building with:
 make
 ```
 
+## :warning: Self upgrade notice :warning:
```

Comment:
I think we can skip all that. The next version will be a major release for completely unrelated reasons anyway.
These can only be installed via re-flashing of the SD card. Therefore, .sh updated can be completely ignored once dpkg is in place. Backwards compatibility is not needed.

Reply:
Can we consider this resolved as script based OTA hasn't been removed and I removed that comment from the README.md as it isn't a limitation at present?